scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -253,6 +253,12 @@ def add_parse_arguments(parser = None) -> argparse.ArgumentParser:
         timeout would be multiplication of test timeout value, board-level timeout multiplier
         and global timeout multiplier (this parameter)""")
 
+    test_xor_subtest.add_argument(
```

Comment:
```suggestion
    parser.add_argument(
```
do not block using that parameter together with --scenario